pyproject.toml

```diff
@@ -1,3 +1,28 @@
+[tool.poetry]
+name = "ms2pip"
+version = "0.0.0-dev"
+description = "MS²PIP: MS² Peak Intensity Prediction"
+authors = ["Sven Degroeve", "Ralf Gabriels", "Kevin Velghe", "Ana Sílvia C. Silva"]
+
 [build-system]
 requires = ["setuptools", "cython", "numpy"]
```

Comment:
So poetry can use setuptools as build-backend?

Reply:
Yes, if you run poetry install it will install ms2pip in the virtualenv using setuptools. Really nice actually, we probably should update the GitHub Actions to use it, as to have a guaranteed environment.